app/src/androidTest/java/com/github/epfl/meili/messages/FirebaseMessageDatabaseAdapterTest.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.epfl.meili.messages
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.internal.runner.junit4.statement.UiThreadStatement
+import com.github.epfl.meili.models.ChatMessage
+import junit.framework.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.*
+
+@RunWith(AndroidJUnit4::class)
+class FirebaseMessageDatabaseAdapterTest {
+    private val MOCK_PATH = "POI/mock-poi"
+    private lateinit var db: FirebaseMessageDatabaseAdapter
+    private val fake_message =
+        ChatMessage("fake_id", "fake_text", "fake_from_id", "fake_to_id", 12345)
+
+    @Before
+    fun initializeDatabase() {
+        UiThreadStatement.runOnUiThread {
+            db = FirebaseMessageDatabaseAdapter(MOCK_PATH)
```

Comment:
What is MOCK_PATH ? Is it used to create a mock database ?

Reply:
I will write it as comment, path is the path of the Firebase database, so yes it is for a mock database